src/creators/MarkerClustererCreator.js

```diff
@@ -0,0 +1,127 @@
+
+export const markerClustererControlledPropTypes = {
+  // http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclustererplus/docs/reference.html
+  averageCenter: PropTypes.bool,
+  calculator: PropTypes.func,
+  clusterClass: PropTypes.string,
+  enableRetinaIcons: PropTypes.bool,
+  gridSize: PropTypes.number,
+  ignoreHidden: PropTypes.bool,
+  imageExtension: PropTypes.string,
+  imagePath: PropTypes.string,
+  imageSizes: PropTypes.array,
+  maxZoom: PropTypes.number,
+  minimumClusterSize: PropTypes.number,
+  styles: PropTypes.array,
+  title: PropTypes.string,
+  zoomOnClick: PropTypes.bool,
+};
+
+export const markerClustererDefaultPropTypes = defaultPropsCreator
+  (markerClustererControlledPropTypes);
+
+const markerClustererUpdaters = {
+  averageCenter       (averageCenter, component) { component.getMarkerClusterer().setAverageCenter(averageCenter); },
+  calculator          (calculator, component) { component.getMarkerClusterer().setCalculator(calculator); },
+  clusterClass        (clusterClass, component) { component.getMarkerClusterer().setClusterClass(clusterClass); },
+  enableRetinaIcons   (enableRetinaIcons, component) { component.getMarkerClusterer().setEnableRetinaIcons(enableRetinaIcons); },
+  gridSize            (gridSize, component) { component.getMarkerClusterer().setGridSize(gridSize); },
+  ignoreHidden        (ignoreHidden, component) { component.getMarkerClusterer().setIgnoreHidden(ignoreHidden); },
+  imageExtension      (imageExtension, component) { component.getMarkerClusterer().setImageExtension(imageExtension); },
+  imagePath           (imagePath, component) { component.getMarkerClusterer().setImagePath(imagePath); },
+  maxZoom             (maxZoom, component) { component.getMarkerClusterer().setMaxZoom(maxZoom); },
+  minimumClusterSize  (minimumClusterSize, component) { component.getMarkerClusterer().setMinimumClusterSize(minimumClusterSize); },
+  styles              (styles, component) { component.getMarkerClusterer().setStyles(styles); },
+  title               (title, component) { component.getMarkerClusterer().setTitle(title); },
+  zoomOnClick         (zoomOnClick, component) { component.getMarkerClusterer().setZoomOnClick(zoomOnClick); },
+};
+
+const {eventPropTypes, registerEvents} = eventHandlerCreator(MarkerClustererEventList);
+
+export const markerClustererEventPropTypes = eventPropTypes;
+
+@componentLifecycleDecorator({
+  registerEvents,
+  instanceMethodName: "getMarkerClusterer",
+  updaters: markerClustererUpdaters,
+})
+export default class MarkerClustererCreator extends Component {
+
+  static propTypes = {
+    mapHolderRef: PropTypes.instanceOf(GoogleMapHolder).isRequired,
+    markerClusterer: PropTypes.object.isRequired,
+  }
+
+  static _createMarkerClusterer (mapHolderRef, markerClustererProps) {
+    const markerClusterer = new MarkerClusterer(mapHolderRef.getMap(), [], composeOptions(markerClustererProps, [
```

Comment:
Do we have a similar GitHub hosted open sourced repo for `MarkerClusterer` just like `InfoBox` does?